app/src/main/java/com/sdp/movemeet/MainActivity.java

```diff
@@ -51,7 +54,7 @@ public void handleRegisterUser() {
     }
 
     public void goToMaps(View view) {
-        startActivity(new Intent(this, MapsActivity.class));
+        startActivity(new Intent(getApplicationContext(), MapsActivity.class));
```

Comment:
What's the difference between using `this` and `getApplicationContext()` here?

Reply:
Yep, you're right. I could simply use "this" haha

Reply:
all interactions related to navigation have been moved to a navigation class to be accessible from the menu and avoid redundancy on all pages. This will be modified when you merge with the main branch